definition/data/sheets/CaseEvent.json

```diff
@@ -10,7 +10,7 @@
   {"CaseTypeID": "MoneyClaimCase", "ID": "InterlocutoryJudgment", "Name": "Interlocutory Judgment Req", "Description": "Interlucatory Judgment Request", "DisplayOrder": 1, "PreConditionState(s)": "open", "PostConditionState": "open", "SecurityClassification": "Public"},
   {"CaseTypeID": "MoneyClaimCase", "ID": "ReferToJudgeByDefendant", "Name": "Defendant referred judge", "Description": "Refer To Judge By Defendant", "DisplayOrder": 1, "PreConditionState(s)": "open", "PostConditionState": "open", "SecurityClassification": "Public"},
   {"CaseTypeID": "MoneyClaimCase", "ID": "ReferToJudgeByClaimant", "Name": "Claimant referred judge", "Description": "Refer To Judge By Claimant", "DisplayOrder": 1, "PreConditionState(s)": "open", "PostConditionState": "open", "SecurityClassification": "Public"},
-  {"CaseTypeID": "MoneyClaimCase", "ID": "CCJRequested", "Name": "CCJ requested", "Description": "CCJ requested", "DisplayOrder": 1, "PreConditionState(s)": "open", "PostConditionState": "open", "SecurityClassification": "Public"},
+  {"CaseTypeID": "MoneyClaimCase", "ID": "CCJRequested", "Name": "CCJ requested", "Description": "CCJ requested", "DisplayOrder": 1, "PreConditionState(s)": "open", "PostConditionState": "*", "SecurityClassification": "Public"},
```

Comment:
`open` -> `*` ?

Reply:
yes, the reason is because to support the feature toggle for the staff decommission from CCD definition side, we are setting the post state to "judgementRequested" else to "open" through claim-store.

Reply:
Ahh! I've not seen/noticed that before. Thanks for explaining